Kitodo-FileManagement/src/main/java/org/kitodo/filemanagement/FileManagement.java

```diff
@@ -157,6 +157,14 @@ public URI rename(URI uri, String newName) throws IOException {
             return null;
         }
 
+        if (isDirectory(uri)) {
+            if (uri.toString().equals(newName + "/")) {
```

Comment:
This did not work on my local system, as `newName` already ends with a `/` character and adding an additional `/` character let the comparing every time be false. Maybe reverting this last change is better as it could be difficult for now to find right solution.

Reply:
I simply extended this check to cover uri with and without trailing slash. Would that work for you?

Reply:
I will test this. Maybe moving the comparing of old and new directory inside the called method as there are the resulting paths created and must not recreated in on other way?

Reply:
Done